app/models/snapshot.rb

```diff
@@ -41,4 +41,8 @@ def restore
     live_object.snapshot_action = "restore"
     live_object.save
   end
+
+  def publicly_visible?
+    object['visible'] && object['discarded_at'].nil?
```

Comment:
Since this only controls when it goes *into* the public API, the visible_from and visible_to dates shouldn't be checked.  E.g. a service may be sent to the public API a month before it is visible, and never updated again

Reply:
The actual filtering was done in https://github.com/wearefuturegov/outpost-api-service/pull/2